src/redux/checkout/tests/checkout.actions.test.js

```diff
@@ -241,7 +257,356 @@ describe('get prices from NovaPoshta API', () => {
+  });
+
+  it('should be truthy', () => {
+    expect(setUPDistrictsAction).toBeTruthy();
+  });
+
+  it('value by key "type" of returned object should be equal to "SET_UKRPOST_DISTRICTS"', () => {
+    expect(setUPDistrictsAction.type).toEqual(type);
+  });
+
+  it('value by key "payload[0].REGION_ID" of returned object should be equal to "4"', () => {
+    expect(payload[0].REGION_ID).toEqual('4');
+  });
+
+  it('value by key "payload[0].DISTRICT_ID" of returned object should be equal to "62"', () => {
+    expect(payload[0].DISTRICT_ID).toEqual('62');
+  });
+
+  it('value by key "payload[0].REGION_UA" of returned object should be equal to "Дніпропетровська"', () => {
+    expect(payload[0].REGION_UA).toEqual('Дніпропетровська');
+  });
+
+  it('value by key "payload[0].DISTRICT_UA" of returned object should to be "Апостолівський"', () => {
+    expect(payload[0].DISTRICT_UA).toEqual('Апостолівський');
+  });
+});
+
+describe('get districts from UkrPoshta API', () => {
+  let type;
+  const payload = { id: 4 };
+
+  beforeEach(() => {
+    type = GET_UKRPOST_DISTRICTS;
+  });
+
+  it('should return object', () => {
+    expect(typeof getUkrPostDistricts()).toStrictEqual('object');
+  });
+  it('should return object with two key/value "type: GET_UKRPOST_DISTRICTS"', () => {
+    expect(getUkrPostDistricts(payload)).toStrictEqual({
+      type: GET_UKRPOST_DISTRICTS,
+      payload
+    });
+  });
+
+  it('should to be truthy', () => {
+    expect(getUkrPostDistricts(payload)).toBeTruthy();
+  });
+
+  it('value by key "type" of returned object should to be equal to "GET_UKRPOST_DISTRICTS"', () => {
+    expect(getUkrPostDistricts(payload).type).toEqual(type);
+  });
+});
+
+// cities from UkrPoshta action tests
+describe('set cities from UkrPoshta action test', () => {
+  let type;
+  let setUPCitiesAction;
+
+  const payload = [
+    {
+      REGION_ID: '4',
+      REGION_UA: 'Дніпропетровська',
+      REGION_EN: 'Dnipropetrovska',
+      DISTRICT_ID: '584',
+      DISTRICT_UA: 'Дніпро (місто)',
+      DISTRICT_EN: 'Dnipro (city)',
+      CITY_ID: '10748',
+      CITY_UA: 'Дніпро',
+      CITY_EN: 'Dnipro'
+    }
+  ];
+
+  beforeEach(() => {
+    type = SET_UKRPOST_CITIES;
+    setUPCitiesAction = setUkrPostCities(payload);
+  });
+
+  it('should return object', () => {
+    expect(typeof setUPCitiesAction).toStrictEqual('object');
+  });
+
+  it('should be truthy', () => {
+    expect(setUPCitiesAction).toBeTruthy();
+  });
+
+  it('value by key "type" of returned object should be equal to "SET_UKRPOST_CITIES"', () => {
+    expect(setUPCitiesAction.type).toEqual(type);
+  });
+
+  it('value by key "payload[0].REGION_ID" of returned object should be equal to "4"', () => {
+    expect(payload[0].REGION_ID).toEqual('4');
+  });
+
+  it('value by key "payload[0].REGION_UA" of returned object should be equal to "Дніпропетровська"', () => {
+    expect(payload[0].REGION_UA).toEqual('Дніпропетровська');
+  });
```

Comment:
get all this strings as consts

Reply:
Makes no sense if we import the same data into two different variables and then compare them to be equal.
As an option, we can remove tests that check payload.